aten/src/ATen/native/native_functions.yaml

```diff
@@ -2399,6 +2399,24 @@
   dispatch:
     DefaultBackend: logaddexp2
 
+- func: xlogy.Tensor(Tensor self, Tensor other) -> Tensor
```

Comment:
Add method, inplace, and out variants, too. 

Unfortunately this will double the number of signatures, since the inplace variant has a different name `xlogy_`.

Reply:
Done.